docs/tutorial-md-to-video-ja.md

```
@@ -0,0 +1,224 @@
+## `/md-to-mulmo` 後の次のステップ
+
+### 内容に対話的に質問する
+
+ドキュメントについて質問できます：
+
+```bash
+npx mulmocast-preprocessor query scripts/your-document/extended_script.json -i
+```
+
+実行例：
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)